errors/custom-error.js

```diff
@@ -0,0 +1,27 @@
+
+// class CustomAPIError extends Error {
+//     constructor(message,statusCode) {
+//         super(message)
+//         this.statusCode = statusCode
+//     }
+// }
+
+// const createCustomError = (msg,statusCode)=>{
+//     return new CustomAPIError(msg,statusCode)
+// }
+
+// module.extends = {createCustomError,CustomAPIError}
+
+class CustomAPIError extends Error {
```

Comment:
Don't make an effort on special errors for now. In the future we will maybe talk about that, but it's much less important